src/sbvr-api/permissions.ts

```diff
@@ -1036,9 +1035,9 @@ const getBoundConstrainedMemoizer = memoizeWeak(
 
 							const table = tables[`${resourceName}$bypass`];
 
-							const permissionsTable = (tables[permissionResourceName] = {
```

Comment:
If you don't assign to `tables[permissionResourceName]` then all these changes to `permissionsTable` are essentially noops?